### ISSUE TITLE
iOS 13 beta Crashing: "Access to UITextField's _placeholderLabel ivar is prohibited. This is an application bug"

### ISSUE THREAD

**Issue by s-gilroy** [devxoul/UITextView-Placeholder#61]
We are using this dependency in our app and as of iOS 13 Beta 2 we are getting crashes because of how the framework uses KVC to access Apple's private APIs.

```
[textfield setValue:[UIColor whiteColor] forKeyPath:@"_placeholderLabel.textColor"];
```
```
"NSGenericException" - reason: "Access to UITextField's _placeholderLabel ivar is prohibited. This is an application bug"
```

Pretty important fix required here: @devxoul

**Comment by Idomo** [comment]
Solved it temporarily with changing 
```
static UIColor *color = nil;
static dispatch_once_t onceToken;
dispatch_once(&onceToken, ^{
      UITextField *textField = [[UITextField alloc] init];
      textField.placeholder = @" ";
      color = [textField valueForKeyPath:@"_placeholderLabel.textColor"];
});
return color;
```
to
```
return [UIColor grayColor];
```

in `UITextView+Placeholder.m` at `defaultPlaceholderColor()`

@devxoul @polonez Still have to find some better fix for this one.

**Comment by zanyzephyr** [comment]
> Solved it temporarily with changing
> 
> ```
> static UIColor *color = nil;
> static dispatch_once_t onceToken;
> dispatch_once(&onceToken, ^{
>       UITextField *textField = [[UITextField alloc] init];
>       textField.placeholder = @" ";
>       color = [textField valueForKeyPath:@"_placeholderLabel.textColor"];
> });
> return color;
> ```
> 
> to
> 
> ```
> return [UIColor grayColor];
> ```
> 
> in `UITextView+Placeholder.m` at `defaultPlaceholderColor()`
> 
> @devxoul @polonez Still have to find some better fix for this one.

I changed it  as you say,  still not work

**Comment by Idomo** [comment]
Hi @zanyzephyr, have you ran “Clean build folder” from the `product`menu?

**Comment by IMHitesh** [comment]
You can fix it using below code 
```

            let iVar = class_getInstanceVariable(UITextField.self, "_placeholderLabel")!
            let placeholderLabel = object_getIvar(self, iVar) as! UILabel
            placeholderLabel.textColor = placeHolderColor
```

**Comment by devxoul** [comment]
It is already fixed in #64.
